Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.xif

```diff
@@ -0,0 +1,28 @@
+[MODEL: dataset=salesforce_test_raw]
```

Comment:
Please provide the real dataset name

Reply:
changed from salesforce_test_raw to salesforce_salesforce_raw